ignite/services/scaffolder/type.go

```diff
@@ -259,24 +258,32 @@ func parseTypeFields(opts addTypeOptions) (field.Fields, error) {
 }
 
 // mapGenerator returns the template generator for a map.
-func mapGenerator(replacer placeholder.Replacer, opts *typed.Options, indexes []string) (*genny.Generator, error) {
+func mapGenerator(replacer placeholder.Replacer, opts *typed.Options, index string) (*genny.Generator, error) {
 	// Parse indexes with the associated type
-	parsedIndexes, err := field.ParseFields(indexes, checkForbiddenTypeIndex)
+	indexSplit := strings.Split(index, ",")
+	if len(indexSplit) > 1 {
```

Comment:
```suggestion
	if strings.Contains(index, ",") {
```

we need to avoid `,` anyway. wdyt?